.github/workflows/ci.yml

```diff
@@ -1,271 +0,0 @@
-name: GitHub Actions
```

Comment:
Why is this being deleted? Will the jobs be moved somewhere else?

Reply:
I've deleted this temporarily to avoid clogging the CI (and will add it back in once I've figured out the CI).  I'm also changing this to draft as I should have done in the first place.